Framework/Geometry/src/Rendering/GluGeometryHandler.cpp

```diff
@@ -97,6 +102,17 @@ void GluGeometryHandler::GetObjectGeom(int &mytype,
       vectors.push_back(Point3);
       vectors.push_back(Point4);
       break;
+    case HEXAHEDRON:
+      mytype = 1;
+      vectors.push_back(Point1);
+      vectors.push_back(Point2);
+      vectors.push_back(Point3);
+      vectors.push_back(Point4);
+      vectors.push_back(Point5);
+      vectors.push_back(Point6);
+      vectors.push_back(Point7);
+      vectors.push_back(Point8);
```

Comment:
I would avoid the `push_back`, as it is not optimal in the scenario that you know how many items you are about to add. I would suggest using `reserve` and then `operator[] (const size_t index)` access i.e. `vectors[i] = Point1`